prometheus/client.go

```diff
@@ -86,17 +86,12 @@ func (in *Client) GetSourceServices(namespace string, servicename string) (map[s
 // GetServiceMetrics returns the Health and Metrics related to the provided service identified by its namespace and service name.
 // Health might be nil when unavailable
 func (in *Client) GetServiceMetrics(namespace string, servicename string, duration time.Duration, step time.Duration,
-	rateInterval string) Metrics {
+	rateInterval string) (Metrics, Health) {
 
-	healthChan, metricsChan := make(chan *Health), make(chan Metrics)
-	go getServiceHealthAsync(in.api, namespace, servicename, healthChan)
-	go getServiceMetricsAsync(in.api, namespace, servicename, duration, step, rateInterval, metricsChan)
+	metrics := getServiceMetrics(in.api, namespace, servicename, duration, step, rateInterval)
```

Comment:
These are no longer done in parallel. Intentional?

Reply:
Yes, I changed my approach from "run everything possible in parallel" to something more reasonable I think, like "run what's make most sense in parallel". Though I'm not yet 100% sure about what _makes most sense_.